mp/conwebsock.py

```diff
@@ -71,23 +71,23 @@ def run(self):
     def __del__(self):
         self.close()
 
-    def on_message(self, ws, message):
```

Comment:
The `websocket-client` README indicates that this argument is still used, why the discrepancy?

Reply:
Maybe documentation is outdated